doc/api/next_api_changes/behaviour.rst

```diff
@@ -155,3 +155,17 @@ support for it will be dropped in a future Matplotlib release.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Previously, keyword arguments were silently ignored when no positional
 arguments were given.
+
+`.Axes.vlines` and `.Axes.hlines` ``color`` parameter will now default to configured rcparams ``lines.color``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Previously the ``color`` parameter in `.Axes.vlines` and `.Axes.hlines` would always default to 'k' if not provided.
+This was inconsistent with the rest of the API and now ``color`` will instead default to the configured rcparam's
+``lines.color`` value, falling back to 'k' if necessary.
```

Comment:
For use in the docs we have a custom sphinx role that will format these nicely (see https://matplotlib.org/devel/documenting_mpl.html?highlight=documenting#rcparams )